release-notes/3.0/preview/3.0.0-preview-known-issues.md

```diff
@@ -23,6 +23,48 @@ This document lists known issues for **.NET Core 3.0 Preview releases** which ma
 
 ## ASP.NET Core
 
+### Preview 6
+
+- **`@attribute` directive does not work in _Imports.razor**
+
+    When adding the `@attribute` directive in *_Imports.razor* the project may fail to build with the following error:
+
+    ```
+    System.IndexOutOfRangeException : Index was outside the bounds of the array.
+    ```
+
+    To workaround this issue add the `@attribute` to the individual .razor files instead of in *_Imports.razor*.
+
+- **Publishing an ASP.NET Core hosted client-side Blazor app to Azure as a self-contained app fails**
+
+    Publishing an ASP.NET Core hosted client-side Blazor app to Azure as a self-contained app fails with the following error:
+
+    ```
+    Error NETSDK1005: Assets file 'C:\Users\user\Source\Repos\WebApplication1\WebApplication1.Client\obj\project.assets.json' doesn't have a target for '.NETStandard,Version=v2.0'. Ensure that restore has run and that you have included 'netstandard2.0' in the TargetFrameworks for your project.
+    ```
+
+    This issue will be addressed in a future release.
+
+- **`@bind-value` on its own results in a build error**
+
+    Writing `<input @bind-value="SOMEVALUE" />` in a component results in the following error:
+
+    ```
+    Error RZ9991: The attribute names could not be inferred from bind attribute '@bind-value'. Bind attributes should be of the form'bind' or 'bind-value' along with their corresponding optional parameters like 'bind-value:event', 'bind:format' etc.
+    ```
+
+    To work around this issue add a `@bind-value:event="SOMEEVENT"` to the input element: `<input @bind-value="SOMEVALUE" @bind-value:event="SOMEEVENT" />`.
+
+- **Using element refs with JavaScript interop in a client-side Blazor apps fails when the IL linker is disabled**
+
+    Using element refs with JavaScript interop in a client-side Blazor apps fails when the IL linker is disabled with the following exception:
+
+    ```
+    System.MissingMethodException: Method not found: !!0& System.Runtime.CompilerServices.Unsafe.Unbox<!0>(object)
+    ```
+
+    To work around this issue enable the IL linker as part of the build.
+
```

Comment:
Should we also mention the known issue that currently you can't use `AuthorizingContent`/`NotAuthorizedContent` on `Router` with server-side Blazor? This will be fixed in p7.

Reply:
Yup, done